tutorial/train_horizon_net/README.md

```diff
@@ -4,18 +4,26 @@ In this tutorial, we will walk you through how to train your own model using pil
 
 ### Step 1: download pre-trained models
 ```bash
+# use -h for more details
 python download_pretrained_ckpt.py
 # or specify download info and output directory
-python download_pretrained_ckpt.py -f {DOWNLOAD_INFO} -o {OUTPUT_DIR}
+python download_pretrained_ckpt.py -f $DOWNLOAD_INFO -o $OUTPUT_DIR
 ```
 Afterwards, you should see all pre-trained models under `mvl_challenge/assets/ckpt`.
 
 ### Step 2: start training HorizonNet
+
+Instead of using the native HorionNet, we provide a wrapper at `mvl_challenge/models/wrapper_horizon_net.py` allowing you to easily train your model on the provided dataset.
+
+This wrapper keeps the overall structure of the original implementation, except for tooking out the post-processing part, which allows estimations of non-Manhattan scenes. You can find all controlable settings and hyperparameters in `train_hn.yaml`, note that parameters marked with `<Required>` should be filled.
```

Comment:
1. typo: tooking
2. "which allows estimations of non-Manhattan scenes"--> Yes, this is true, but the real reason we want to avoid post-processing is to truly evaluate HN model.  
3. note that parameters marked with `<Required>` should be filled. --> Add ref to this part in the code https://github.com/mvlchallenge/mvl_toolkit/blob/aee0c1a718419aca6eb99fad75bd7689c5d42e5d/tutorials/train_horizon_net/train_hn.py#L17-L25

Reply:
1. fixed
2. could you help me with that? @EnriqueSolarte 
3. not sure what you mean. could you explain more or changing directly is welcome.